Assignment 3_10-30.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Assignment 3: K Means Clustering"
+---
+
+In this assignment we will be applying the K-means clustering algorithm we looked at in class. At the following link you can find a description of K-means:
+
+https://www.cs.uic.edu/~wilkinson/Applets/cluster.html
+
+
+```{r}
+library()
+```
+
+Now, upload the file "Class_Motivation.csv" from the Assignment 3 Repository as a data frame called "K1""
+```{r}
+
+K1 <- read.csv("Class_Motivation.csv", header=TRUE, sep = ",")
+```
+
+This file contains the self-reported motivation scores for a class over five weeks. We are going to look for patterns in motivation over this time and sort people into clusters based on those patterns.
+
+But before we do that, we will need to manipulate the data frame into a structure that can be analyzed by our clustering algorithm.
+
+The algorithm will treat each row as a value belonging to a person, so we need to remove the id variable.
+
+```{r}
+
+K2 <- select(K1, -"id")
+View(K2)
+
+```
+
+It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
+
+##Role of missing values in clustering: What are the implication of removing missing values? In removing missing varibles, you are reducing your sample size and introducing standard error.
```

Comment:
I'm not sure what you mean by standard error? Can you be more specific about how missing values might impact the clustering specifically?